### DISCUSSION TITLE
Branch protection rules

### DISCUSSION THREAD

**Discussion by nikomatsakis** [lalrpop/lalrpop#719] [General]
The current rules seem to always require being overridden, even if there are approving reviews etc?

**Comment by youknowone** [comment]
The rule is all test passed + at least 1 (non-self) PR approve

**Reply by youknowone** [reply]
<img width="798" alt="image" src="https://user-images.githubusercontent.com/69878/227715930-9cc974dc-f6ea-4b03-ad6f-c9c34204b458.png">

**Reply by yannham** [reply]
Then the protection has been blocking because of the mdbook lint failure? If that's the case, it's a temporary (broken) state, and I personally find the current policy reasonable, for what it's worth.

**Reply by nikomatsakis** [reply]
I agree this is a reasonable rule, but I thought I was seeing blocks even when a PR had been approved and was passing tests. Let's keep an eye on it.

**Reply by youknowone** [reply]
Before mdbook build is fixed, yes, that was one blocker.
Maybe it could be confused when we merged mdbook patch but didn't rebase other PRs yet.
Let me try to fix it if it happens again.

**Reply by youknowone** [reply]
I found I actually misconfigured it. I replaced "Lock branch" to "Restrict who can push", then now it seems working fine.
